client/components/preferences.js

```diff
@@ -17,8 +17,7 @@ const useStyles = makeStyles({
 });
 
 // exports our ContinuousSlider
-export default function ContinuousSlider(props) {
-  const { currentUser, currentId } = props;
+export default function ContinuousSlider({ currentUser }) {
```

Comment:
Change this to a function expression and move the export default to the end of the file

Reply:
Changed it